db/controllers/users.controllers.js

```diff
@@ -0,0 +1,10 @@
+const { fetchUsers } = require("../models/users.models")
+
+
+exports.getUsers = (request, response, next) => {
+
+    fetchUsers().then((users) => {
+    response.status(200).send({users: users})
+    })
```

Comment:
There should still be a catch block here just in case something unexpected goes wrong